src/easylink/implementation_metadata.yaml

```diff
@@ -204,13 +204,24 @@ dummy_step_1_for_output_dir_example_default:
   - step_1_for_output_dir_example
   image_path: /mnt/team/simulation_science/priv/engineering/er_ecosystem/images/zmbc/dummy_step_1_for_output_dir_example.sif
   script_cmd: python /dummy_step_1_for_output_dir_example.py
-  # leave outputs out for testing purposes
```

Comment:
Comments get removed from this tool evidently. I guess when you load a yaml (which this does), it removes the comments